nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/utils/SNMPUtils.java

```diff
@@ -183,86 +192,150 @@ private static void addGetOidValues(Map<String, String> attributes, Object vecto
 
     /**
      * Method to add {@link FlowFile} attributes from a {@link VariableBinding}
+     *
      * @param variableBinding {@link VariableBinding}
-     * @param attributes {@link FlowFile} attributes to update
+     * @param attributes      {@link FlowFile} attributes to update
      */
     private static void addAttributeFromVariable(VariableBinding variableBinding, Map<String, String> attributes) {
         attributes.put(SNMP_PROP_PREFIX + variableBinding.getOid() + SNMP_PROP_DELIMITER + variableBinding.getVariable().getSyntax(), variableBinding.getVariable().toString());
     }
 
     /**
      * Will validate if provided name corresponds to valid SNMP property.
+     *
      * @param name the name of the property
      * @return 'true' if valid otherwise 'false'
      */
     public static boolean isValidSnmpPropertyName(String name) {
-        return propertyNames.contains(name);
+        return PROPERTY_NAMES.contains(name);
     }
 
     /**
      * Method to extract property name from given {@link Method}
+     *
      * @param method method
      * @return property name
      */
     private static String extractPropertyNameFromMethod(Method method) {
-        char c[] = method.getName().substring(3).toCharArray();
+        char[] c = method.getName().substring(3).toCharArray();
         c[0] = Character.toLowerCase(c[0]);
         return SNMP_PROP_PREFIX + new String(c);
     }
 
     /**
-     * Method to return the private protocol given the property
+     * Method to return the private protocol given the property.
+     *
      * @param privProtocol property
      * @return protocol
      */
     public static OID getPriv(String privProtocol) {
         switch (privProtocol) {
-        case "DES":
-            return PrivDES.ID;
-        case "3DES":
-            return Priv3DES.ID;
-        case "AES128":
-            return PrivAES128.ID;
-        case "AES192":
-            return PrivAES192.ID;
-        case "AES256":
-            return PrivAES256.ID;
-        default:
-            return null;
+            case "DES":
```

Comment:
As these values are also used as allowable property values, it would be helpful to define an enum that could be reused in both places.  The enum could also be used to include the associated OID which could remove the need for this method.